UnityProject/Assets/Scripts/Objects/ExosuitFabricator.cs

```diff
@@ -62,10 +57,17 @@ private void Awake()
 
 		public void EnsureInit()
 		{
-			SyncSprite(ExosuitFabricatorState.Idle, ExosuitFabricatorState.Idle);
+			spriteHandler = GetComponentInChildren<SpriteHandler>();
 			materialStorage = this.GetComponent<MaterialStorage>();
 		}
 
+		private void Start()
+		{
+			if(CustomNetworkManager.IsServer == false) return;
```

Comment:
I remember there was a Server version of this event method, so you get rid of the if.